modules/olmv1-installing-an-operator.adoc

```diff
@@ -7,9 +7,7 @@
 [id="olmv1-installing-an-operator_{context}"]
 = Installing a cluster extension from a catalog
 
-{olmv1-first} supports installing cluster extensions, including {olmv0} Operators via the `registry+v1` bundle format, that are scoped to the cluster. You can install an extension from a catalog by creating a custom resource (CR) and applying it to the cluster.
-
-include::snippets/olmv1-tp-extension-support.adoc[]
+You can install an extension from a catalog by creating a custom resource (CR) and applying it to the cluster. {olmv1-first} supports installing cluster extensions, including {olmv0} Operators via the `registry+v1` bundle format, that are scoped to the cluster. For more information, see _Supported extensions_.
```

Comment:
So in docs are we normally referring to them as `cluster extension`, lower case, or `Cluster Extension` or ClusterExtension? I worry about `cluster extension`, lower case, because it doesn't seem like a special thing, an object I am dealing with on the cluster.

Reply:
If we are referring to the specific API object, the we use `ClusterExtension`. If we are referring to a more generic reference, we use "cluster extension". So, since we aren't referring to the `ClusterExtension` API, the right usage downstream is to use "cluster extension" or "extension". Cluster extensions and extensions aren't special. These terms replace the _Operator_ term in legacy OLM docs.

Reply:
okay makes sense